### DISCUSSION TITLE
windows version / wsl version / kernel version

### DISCUSSION THREAD

**Discussion by tweetysat** [microsoft/WSL#7718] [Q&A]
I don't understand very well what's the link between windows build version and wsl version/kernel version.

In my case
windows 10 21H2 build 19044.1348
wsl --status -> 4.19.128
(gcm wsl).Version -> 10     0      19041  1320

Looking at https://docs.microsoft.com/en-us/windows/wsl/kernel-release-notes and https://docs.microsoft.com/en-us/windows/wsl/release-notes, it seems I'm not up-to-date.

Can I update to the latest kernel (5.10.60.1) on any windows build or is it linked ?
Same question for wsl (21364) ?

The kernel can be updated with wsl --update.  Right ? 
It can be automatic with windows update ?
And for wsl ?  It linked to windows or it can be updated apart ?

I'm working on a corporate laptop without admin rights.  I suppose there is no way to update ?

Thanks.

**Comment by sliekens** [comment]
I'm wondering the same thing. Looking for a way to relate the release notes to the version I'm using, but it seems not possible.
